RetailCoder.VBE/UI/RubberduckUI.fr.resx

```diff
@@ -1407,10 +1159,7 @@ Support localisation WPF par Grant Frisken</value>
     <value>Rubberduck</value>
   </data>
   <data name="AboutWindow_BlogsHeader" xml:space="preserve">
-    <value>Blogues:</value>
-  </data>
-  <data name="SourceControl_UnpublishBranchButton" xml:space="preserve">
-    <value>Dépublier</value>
+    <value>Blogues :</value>
```

Comment:
`SourceControl*` keys are definitely stale now. ...also (irrelevant now, but I'm curious about this one) wouldn't "blogues" be a translation for "blog"? TBH translating source control terminology was a massive headache :smile:

Reply:
I only added the space before the semi-colon. 
I wouldn't translate blog into blogue, but left the proposed translation I found.  I must have taken an too old version of the file.
IMHO "Dépublier" is a better translation.